pkg/ffresty/config.go

```diff
@@ -67,6 +67,8 @@ const (
 	HTTPIdleTimeout = "idleTimeout"
 	// HTTPMaxIdleConns the max number of idle connections to hold pooled
 	HTTPMaxIdleConns = "maxIdleConns"
+	// HTTPRPS the max number of request to submit per second, default value is 0, which turns off the RPS control
+	HTTPRPS = "rps"
```

Comment:
Would prefer more explicit configuration name
```suggestion
	HTTPRPS = "maxRequestsPerSecond"
```

Reply:
Is it because it's too short, or developers using ffresty wouldn't understand what RPS is (like they would do for URL/TLS)?  `rps` is a common attribute for most ingress configurations, that's the reason i keep it consistent.